blacklight-cornell/config/initializers/backlight_custom_helper_loader.rb

```diff
@@ -0,0 +1,13 @@
+#########################################
+## Override Blacklight Helper Methods  ##
+#########################################
+# ==============================================================================
+# This prepends any custom helper methods you have defined with the associated
+# Blacklight helper module. You can use 'super' to fallback on the blacklight
+# logic if you want to extend the existing behavior.
+#
+# *** Only Overrides Blacklight Helper Methods You Add ***
+# ------------------------------------------------------------------------------
+Rails.application.config.to_prepare do
+  Blacklight::SearchHistoryConstraintsHelperBehavior.prepend(CornellSearchHistoryConstraintsHelper)
```

Comment:
Nice. We use this prepend pattern in a few other repos (usda-esmis-sufia, exhibits-library-cornell-edu), but we add all those prepends in application.rb instead of its own initializer - I like this better!